src/m365/teams/commands/channel/channel-remove.ts

```diff
@@ -140,14 +130,52 @@ class TeamsChannelRemoveCommand extends GraphCommand {
         type: 'confirm',
         name: 'continue',
         default: false,
-        message: `Are you sure you want to remove the channel ${channelName} from team ${args.options.teamId}?`
+        message: `Are you sure you want to remove the channel ${channelName}?`
```

Comment:
Is there a reason you removed 'from team `${args.options.teamId}`? 
I think this is still important information on a confirm. Let's keep it in (showing the team name or id)